tox.ini

```diff
@@ -20,6 +20,7 @@ commands =
   # we use npm/mocha to perform testing instead of pytest, mainly because
   # ajv-cli does not allow passing options we need.
   # {envpython} -m pytest
+  npm run test
```

Comment:
Why is this included with the schema removal?

Reply:
Safety. I wanted to see that testing passes. Less than ideal but with current state of affairs...